### ISSUE TITLE
Resizing the 3bot on the 3bot deployer page immediately starts the 3bot

### ISSUE THREAD

**Issue by weynandkuijpers** [threefoldtech/js-sdk#2568]
Resizing the 3bot (a stopped one) on the 3bot deployer page results in the 3bot being started after you have provided input with regards to resizing the 3bot.  I wanted to scale down the 3bot to make it fit on a particular (small) server in testnet (OGFarm).

My suggestion is to add a step after providing the new size information and ask to start the 3bot or not.  In this case, I wanted to resize "downsize" my 3bot to firs on a small node and then move the 3bot to that small node (is i used to run on a large node on freefarm.  Now I need to stop the 3bot twice to make that action work.

So:
- stop node to resize it
- resise node 
- ask to move node or start node
- move node (it requested)
- start node (if requested)
- no nothing more than resizing a stopped now if previous two questions are answered negative

**Comment by xmonader** [comment]
I don't think that valid, 3Bot needs to be running to validate the success of the migration / or resize. too much hassle to have right now.
